test/request-handler.test.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-env mocha */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+
+import assert from 'assert';
+import dotenv from 'dotenv';
+import { stub } from 'sinon';
+import { RequestHandler } from '../src/index.js';
+
+dotenv.config();
+
+const MOCK_REQUEST = new Request('http://localhost', { method: 'POST' });
+function mockContext(event) {
+  return {
+    invocation: {
+      event: event || {},
+    },
+    env: process.env,
+  };
+}
+
+function mockSqsRecord(action) {
+  return {
+    messageId: 'af88e691-c3a6-4b46-b4d2-1c897b41b600',
+    receiptHandle:
+      'AQEBJCLTpWgDm+oaeBAlSKWumzIoFRHeJglHCwWEfJANgc7GSWQBcYTiLPfbO1IuxAIkJagUIEkqgmszqnj2a7hLZjoIcv0AWCQfL0tmje/hhnDWYKdQmrUmfITdPDIg49XI+n+Ub/gKjXEy3VvunLsp0bxuF33OCsR8+N0Skff+U+zan+42GcHtn8lacm6ZQIF9msoFxszourA+zpJ/DJ1DTMlEpr9cSPxa6nsbg7JHOOwBzWknn7d3Zkimuo/J3shMyb+4fBYFRNpzXt9o9l8rfQpi9JZDwGIFRqDYFvpI0Emqv9ke1V2uBAJPiiGS0h1MIKO6dZZ/ejfWAR0Rug3zMEH9SEa6N+hT4gF5Pu2IN6WmcRhE4sh0jW/ImAAunuIo/OZ1FhNjqp+keK3AvBiPiQ==',
```

Comment:
just want to confirm this is mocked data as it looks a little too real :)

Reply:
Good point, actually mocking.